src/input/manager.cc

```diff
@@ -64,7 +64,7 @@ Manager::pressed(int key) {
   if (m_textInput != NULL)
     m_textInput->pressed(key);
   else
-    std::find_if(rbegin(), rend(), std::bind2nd(std::mem_fun(&Bindings::pressed), key));
+    std::for_each(rbegin(), rend(), [&key](Bindings* bind) { bind->pressed(key); });
```

Comment:
These do not do the same thing, `find_if` stops when true.

Reply:
Reverted to `find_if`.